core/src/v2/schema.rs

```diff
@@ -312,6 +316,10 @@ impl<T> Apiv2Schema for Option<T> {
     default fn security_scheme() -> Option<SecurityScheme> {
         None
     }
+
+    fn is_required() -> bool {
```

Comment:
Can we make this a constant? It's a primitive value after all.

Reply:
Done :)